README.md

```diff
@@ -151,6 +151,24 @@ script:
 
 The monitoring dashboard is accessible at `http://kubernetes-master-1.kubernetes-playground.local/monitoring/cluster`
 
+### Kites experiments
+
+#### How to deploy the net-test-dev DaemonSet
+
+On the Master node, run:
+`kubectl apply -f /vagrant/kubernetes/kites/net-test-dev_ds.yaml`
+
+Check that the pods created are in the "Running" state with:
```

Comment:
What should I look for to check that everything was deployed successfully?

Reply:
Also, we're using DaemonSets, so we should also check if daemonsets were deployed successfully